src/main/telemetry/smartport.c

```diff
@@ -452,7 +448,7 @@ void handleSmartPortTelemetry(void)
 #ifdef GPS
             case FSSP_DATAID_GPS_ALT    :
                 if (sensors(SENSOR_GPS) && STATE(GPS_FIX)) {
-                    smartPortSendPackage(id, GPS_altitude * 1000); // given in 0.1m , requested in 100 = 1m
+                    smartPortSendPackage(id, GPS_altitude * 100); // given in 0.1m , requested in 10 = 1m
```

Comment:
from opentx: 
https://github.com/opentx/opentx/blob/next/radio/src/telemetry/frsky_sport.cpp#L80
` { GPS_ALT_FIRST_ID, GPS_ALT_LAST_ID, ZSTR_GPSALT, UNIT_METERS, 2 },`

So original comment should be correct and sent value should be `GPS_altitude * 10` ...